--- pkg/proton-ge-custom/PKGBUILD 2024-03-25 18:23:50.535110577 -0400
+++ pkg/proton-ge-custom/PKGBUILD	2024-03-25 23:42:57.560063892 -0400
@@ -182,8 +182,8 @@ build() {
     local -a split=($CFLAGS)
     local -A flags
     for opt in "${split[@]}"; do flags["${opt%%=*}"]="${opt##*=}"; done
-    local march="${flags["-march"]:-nocona}"
-    local mtune="generic" #"${flags["-mtune"]:-core-avx2}"
+    local march='native'
+    local mtune='nocona'
 
     CFLAGS="-O3 -march=$march -mtune=$mtune -pipe -fno-semantic-interposition"
     CXXFLAGS="-O3 -march=$march -mtune=$mtune -pipe -fno-semantic-interposition"
